stream-chat-android-ui-components/src/main/res/values/strings.xml

```diff
@@ -73,4 +73,8 @@
         <item quantity="one">%d Message Reaction</item>
         <item quantity="other">%d Message Reactions</item>
     </plurals>
+
+    <!-- Message Preview -->
+    <string name="stream_ui_message_file">File: %s</string>
+    <string name="stream_ui_message_sender_title_in_channel">&lt;b&gt;%1$s&lt;/b&gt; in &lt;b&gt;%2$s&lt;/b&gt;</string>
```

Comment:
I'm so sorry 😄

Reply:
Maybe CDATA?

```
<![CDATA[
    Since this is a CDATA section
    I can use all sorts of reserved characters
    like > < " and &
    or write things like
    <foo></bar>
    but my document is still well formed!
]]>
```

Reply:
Not sure if this would be much of an improvement, both feel and look kinda ugly and hacky 😄